File: android/versioned-abis/expoview-abi31_0_0/src/main/java/abi31_0_0/expo/modules/analytics/segment/SegmentPackage.java
package abi31_0_0.expo.modules.analytics.segment;

import android.content.Context;

import java.util.Collections;
import java.util.List;

import abi31_0_0.expo.core.BasePackage;
import abi31_0_0.expo.core.ExportedModule;

public class SegmentPackage extends BasePackage {
  @Override
  public List<ExportedModule> createExportedModules(Context context) {
    return Collections.singletonList((ExportedModule) new SegmentModule(context));
  }
}

File: android/versioned-abis/expoview-abi44_0_0/src/main/java/abi44_0_0/expo/modules/analytics/segment/SegmentPackage.kt
package abi44_0_0.expo.modules.analytics.segment

import android.content.Context
import abi44_0_0.expo.modules.core.BasePackage
import abi44_0_0.expo.modules.core.ExportedModule

class SegmentPackage : BasePackage() {
  override fun createExportedModules(context: Context): List<ExportedModule> {
    return listOf(SegmentModule(context))
  }
}
